app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,22 @@
+class LikesController < ApplicationController
+  def create
+    new_like = current_user.likes.build
+    new_like.post_id = params[:post_id]
+    new_like.user_id = current_user.id
+
+    check_liked = Like.find(current_user.id)
```

Comment:
- [ ] There seems to be a problem when I try to like a post. From your code, I can see that you are trying to find a Like data with row 1 `(current_user.id = 1)`, but for the first user of your app and the first like, there is no such info. Kindly fix this.

  ![clint-likes](https://user-images.githubusercontent.com/36542986/166810037-43e11c0b-17cc-4582-9bed-f9ea22d35f6f.jpg)

  ## SUGGESTION
  I think you should use the `find_by` method instead to look for a Like item where `user_id = current_user.id`, and `post_id = new_like.post_id` like so:
  ```
  check_liked = Like.find_by(user_id: current_user.id, post_id: new_like.post_id)
  ```

Reply:
Thank you for calling my attention to this, It was a slight oversight. I have implemented the required changes